.devcontainer/Dockerfile.dev

```diff
@@ -0,0 +1,53 @@
+FROM debian:buster
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils dialog libicu63 libssl1.1 2>&1 \
+    #
+    # Verify wget, jq, unzip, git and process tools installed
+    && apt-get -y install git iproute2 procps wget unzip jq \
+    #
+    # Upgrade existing packages
+    && apt-get -y upgrade \
+    #
+    # Install Docker CE CLI
+    && apt-get install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common lsb-release \
+    && curl -fsSL https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]')/gpg | (OUT=$(apt-key add - 2>&1) || echo $OUT) \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce-cli \
+    #
+    # Install dotnet core 5
+    && cd /tmp \
+    && wget https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -O packages-microsoft-prod.deb \
+    && dpkg -i packages-microsoft-prod.deb \
+    && apt-get update \
+    && apt-get install -y apt-transport-https \
+    && apt-get update \
+    && apt-get install -y dotnet-sdk-5.0 \
+    #
+    # Install node
+    && curl -sL https://deb.nodesource.com/setup_12.x | bash - \
+    && apt install -y nodejs \
+    #
+    # Install npm-check-updates tool globally
+    && npm install -g npm-check-updates \
+    #
+    # Install jest testing framework globally
+    && npm install -g jest \
```

Comment:
I think we're only adding jest globally, right? I wonder having it only in the `package.json` isn't enough?

we should be able to use it with an `npx` command, like, `npx jest --help`, etc.

Reply:
This is meant to be jest-cli globally, updating